task/06-conditions-n-loops-tasks.js

```diff
@@ -63,9 +79,13 @@ function getFactorial(n) {
  *   -1,1  =>  0  ( = -1 + 0 + 1 )
  */
 function getSumBetweenNumbers(n1, n2) {
-    throw new Error('Not implemented');
-}
+    let count=0;
+    for(let i=n1; i<=n2;i++){
```

Comment:
spaces between operators 
here and below, please https://javascript.info/coding-style